tldw_Server_API/app/core/MCP_unified/tests/test_prometheus_validation_metrics.py

```
@@ -138,9 +102,9 @@ async def check_permission(self, *args, **kwargs):
         json=bad,
         headers={"Authorization": f"Bearer {token}", "X-Forwarded-For": "127.0.0.1"},
     )
-    assert r1.status_code == 200
+    assert r1.status_code == 200  # nosec B101
     body1 = r1.json()
-    assert isinstance(body1, dict) and body1.get("error") is not None
+    assert isinstance(body1, dict) and body1.get("error") is not None  # nosec B101
```

Ruff (0.15.9)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)